### PR TITLE
Add Jobs API to invoke Data Model APIs for OLAP and OLTP sync

### PR DESCRIPTION
**Pull request by tuxdna**
For: https://github.com/fabric8-analytics/fabric8-analytics-jobs/issues/239

### CODE REVIEW

File: f8a_jobs/graph_sync.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import f8a_jobs.defaults as configuration
+import requests
+import traceback
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def _api_call(url, params={}):
```
Comment (jpopelka) [inline f8a_jobs/graph_sync.py]: Using `{}` as a default argument isn't a good idea, see http://docs.python-guide.org/en/latest/writing/gotchas

Use something like:
```python
def _api_call(url, params=None):
    params = params or {}
```

File: f8a_jobs/graph_sync.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    try:
+        logger.info("API Call for url: %s, params: %s" % (url, params))
+        r = requests.get(url, params=params)
+        if r is None:
```
Comment (jpopelka) [inline f8a_jobs/graph_sync.py]: You might rather do `r.raise_for_status()`

File: f8a_jobs/graph_sync.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            logger.error("Returned response is: %s" % r)
+            raise Exception("Empty response found")
+
+        result = {"data": r.json()}
+    except Exception:
```
Comment (jpopelka) [inline f8a_jobs/graph_sync.py]: `requests.exceptions.HTTPError`

File: f8a_jobs/graph_sync.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        logger.error(traceback.format_exc())
+        result = {"error": "Failed to retrieve data from Data Model Importer backend"}
+    return result
+
+
+def fetch_pending(params={}):
+    """Invoke Pending Graph Sync APIs for given parameters."""
+    url = "%s%s" % (configuration.DATA_IMPORTER_ENDPOINT, "/api/v1/pending")
```
Comment (jpopelka) [inline f8a_jobs/graph_sync.py]: Or simply `url = configuration.DATA_IMPORTER_ENDPOINT + "/api/v1/pending"`

Or more sophisticated
```python
from urllib.parse import urljoin
url = urljoin(configuration.DATA_IMPORTER_ENDPOINT, "/api/v1/pending")
```

**Comment by tuxdna** [comment]
@jpopelka Thanks for the review. Let me update the code and push again.

**Comment by tuxdna** [comment]
@jpopelka Updated the code as per review comments.

**Pull request merged**